resources/subgraphs/subgraph-data.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 1
+title: Subgraph Data Introduction
+---
+
+# UMA Subgraph Introduction
+
+This section explains the UMA subgraph and how to interact with it. The UMA subgraph indexes data from UMA contracts over time. It organizes data about tokenholders, contracts, DVM requests, voting, and more. The subgraph updates any time a transaction is made on UMA. The subgraph runs on [The Graph](https://thegraph.com/) protocol’s hosted service and can be openly queried.
+
+UMA has a GraphQL API Endpoint hosted by [The Graph](https://thegraph.com/docs/about/introduction#what-the-graph-is) called a subgraph for indexing and organizing data from the smart contracts. The schema of GraphQL elements available is defined in [`the subgraphs repo` ](https://github.com/UMAprotocol/subgraphs/).
+
+## Ethereum Mainnet
+
+[Creating an API Key Video Tutorial](https://www.youtube.com/watch?v=UrfIpm-Vlgs)
+
+- [Explorer Page](https://thegraph.com/explorer/subgraph?id=FVmuv3TndQDNd2BWARV8Y27yuKKukryKXPzvAS5E7htC&view=Overview)
+- Graphql Endpoint: https://gateway.thegraph.com/api/[api-key]/subgraphs/id//41LCrgtCNBQyDiVVyZEuPxbvkBH9BxxLU3nEZst77V8o

```

Comment:
This is a broken link. Is an api-key needed to access?

Reply:
The end point link is really to copy and paste into websites like https://graphiql-online.com/graphiql.  When you plug in as end point it allows you to develope different queries for UMA